core/src/main/java/org/infinispan/commands/write/WriteCommand.java

```diff
@@ -64,4 +68,16 @@ default boolean isWriteOnly() {
       return false;
    }
 
+   default void updateMetadata(CacheEntry entry, Metadata providedMetadata) {
```

Comment:
can be a static method. It will not be implemented neither access outside the write command.